pkg/server/server.go

```diff
@@ -80,18 +84,11 @@ func New(cfg config.Config, stopCh <-chan struct{}) *Server {
 		Config: cfg,
 	}
 
-	mappers, err := BuildMapperChain(cfg)
+	backendMapper, err := BuildMapperChain(cfg, cfg.BackendMode)
```

Comment:
I think if DynamicBackendModePath is set then server startup should block until the server loads backend modes from the file for the first time instead of using cfg.BackendMode

Right now there is a brief period in which the server uses cfg.BackendMode until it loads backend modes from DynamicBackendModePath. I would prefer if I set cfg.DynamicBackendModePath that the server use cfg.DynamicBackendModePath exclusively, and ignore cfg.BackendMode, to avoid any potential race/misconfiguration.

For example let's say I leave BackendMode = MountedFile and assume that the server uses DynamicBackendModePath=DynamicFile instead. But if somebody accidentally deletes DynamicBackendModePath then the next time the server starts it will use BackendMode=MountedFile when that is not my intention.

Reply:
Thank you for the comments! It is designed this way by the moment. cfg provides the default backend mode, and the dynamic file might not be available during startup.

Reply:
ok thx. Please add this to the comment for `DynamicBackendModePath`. 

The other edge case is if DynamicBackendModePath gets deleted then the server just keeps using the last DynamicBackendModePath it read until it restarts since CallBackForFileDeletion is not implemented. Should it fallback to cfg.BackendMode?

Reply:
this is a good point. updated to fallback to cfg.BackendMode